tlf/id.go

```diff
@@ -88,6 +88,43 @@ func TypeFromFolderType(ft keybase1.FolderType) Type {
 	}
 }
 
+// KeyingType represents a TLF keying mode. It normally have the same values
+// as Type.
+type KeyingType Type
+
+const (
+	// UnknownKeying is a placeholder type for when TLF keying mode is unknown.
+	UnknownKeying = KeyingType(Unknown)
+	// PrivateKeying specifies the  TLF keying mode used in classic private TLF
+	// folders.
```

Comment:
"TLF folders" is redundant, since the "F" stands for "folder".  Can just be "TLFs" (here and below).